scripts/lua.sh

```diff
@@ -13,17 +13,27 @@ fi
 
 # LUA_T= and LUAC_T= disable building lua & luac
 # -Dgetlocaledecpoint()=('.') fixes bionic missing decimal_point in localeconv
-make CC="arm-linux-androideabi-gcc -Dgetlocaledecpoint\(\)=\(\'.\'\)" PLAT=linux LUA_T= LUAC_T= -j6
+
```

Comment:
the comment belongs to the command below so it'd make sense not having an empty line inbetween imo